### PR TITLE
Fix: fixed bugs while flushing

### PR DESCRIPTION
**Pull request by hmoog**

### CODE REVIEW

File: objectstorage/batch_writer.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 
 const (
 	BATCH_WRITER_QUEUE_SIZE    = BATCH_WRITER_BATCH_SIZE
-	BATCH_WRITER_BATCH_SIZE    = 1024
+	BATCH_WRITER_BATCH_SIZE    = 10240
```
Comment (alexsporn) [inline objectstorage/batch_writer.go:17]: was this change intentional?

Comment (hmoog) [reply objectstorage/batch_writer.go:17]: yes

File: objectstorage/cached_object.go [hunk 2]
```diff
@@ -88,7 +88,11 @@ func (cachedObject *CachedObjectImpl) Release(force ...bool) {
 				}
 			})))
 		} else {
-			cachedObject.objectStorage.options.batchedWriterInstance.batchWrite(cachedObject)
+			// only force release if there is no timer running, so that objects that landed in the cache through normal
```
Comment (muXxer) [inline objectstorage/cached_object.go:91]: :heart_eyes:
